aiida/cmdline/commands/cmd_code.py

```diff
@@ -245,7 +245,7 @@ def code_list(computer, input_plugin, all_entries, all_users, show_owner):
 
     qb_computer_filters = dict()
     if computer is not None:
-        qb_computer_filters['name'] = computer.name
+        qb_computer_filters['name'] = computer.lbael
```

Comment:
```suggestion
        qb_computer_filters['name'] = computer.label
```

Reply:
not sure whether this is also in other places (I guess not?) - a quick search should show